gsitemap-cron.php

```diff
@@ -59,4 +59,5 @@
     
     /* Create the Google sitemap's files */
     $gsitemap->createSitemap((int) $id_shop);
+    $gsitemap->indentXml();
```

Comment:
Why not format the XML directly using `formatXML()` in `createSitemap()`?

Reply:
Because this files are wrote many times, so i want to format once the files is finished.

Reply:
@eternoendless Indeed in order to write properly indented XML files directly, we need to modify functions like [recursiveSitemapCreator](https://github.com/PrestaShop/gsitemap/blob/master/gsitemap.php#L655) that build XML without using dedicated php functions.